server/schedule/operator_controller.go

```diff
@@ -973,3 +972,30 @@ func (oc *OperatorController) RemoveStoreLimit(storeID uint64) {
 	}
 	delete(oc.storesLimit, storeID)
 }
+
+// CollectStoreLimitMetrics collects the metrics about store limit
+func (oc *OperatorController) CollectStoreLimitMetrics() {
```

Comment:
Where is the new added function called?

Reply:
> Where is the new added function called?

It is called in server/cluster/cluster.go:115.